src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sumRentals = days * 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const rentalplusseven = 50;
+  const rentalplusthree = 20;
```

Comment:
```suggestion
  const rentalPlusThree = 20;
```
same here
and it is better to use more semantic names (e.g `smallDiscount` / `largeDiscount`)